backend/module/chaos/serverexperimentation/server_experimentation.go

```diff
@@ -0,0 +1,84 @@
+package serverexperimentation
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/any"
+	"github.com/uber-go/tally"
+	"go.uber.org/zap"
+
+	experimentation "github.com/lyft/clutch/backend/api/chaos/experimentation/v1"
+	serverexperimentation "github.com/lyft/clutch/backend/api/chaos/serverexperimentation/v1"
+	"github.com/lyft/clutch/backend/module"
+	"github.com/lyft/clutch/backend/service"
+	"github.com/lyft/clutch/backend/service/chaos/experimentation/experimentstore"
+)
+
+const (
+	Name = "clutch.module.chaos.serverexperimentation"
+)
+
+type Service struct {
+	storer experimentstore.Storer
+}
+
+// New instantiates a Service object.
+func New(_ *any.Any, logger *zap.Logger, scope tally.Scope) (module.Module, error) {
+	store, ok := service.Registry[experimentstore.Name]
+	if !ok {
+		return nil, errors.New("could not find experiment store service")
+	}
+
+	storer, ok := store.(experimentstore.Storer)
+	if !ok {
+		return nil, errors.New("service was not the correct type")
+	}
+
+	return &Service{
+		storer: storer,
+	}, nil
+}
+
+func (s *Service) Register(r module.Registrar) error {
+	s.storer.RegisterTransformation("type.googleapis.com/clutch.chaos.serverexperimentation.v1.TestConfig", transform)
+	return nil
+}
```

Comment:
Handle error for `RegisterTransformation`

Reply:
RegisterTransformation does not return an error

Reply:
Yeah I misread the signature for that function. Will `Register` ever return an error then?

Reply:
Made this function to return an `error` in case we need this in the future (it's possible). It's good to have this method to be future proof since it will be used by external packages.